{
  "check": "at least one",
  "comment": "python38-setuptools is not installed",
  "id": "oval:com.redhat.unaffected:tst:202015801034",
  "version": "635",
  "check_existence": "none_exist",
  "object": {
    "object_ref": "oval:com.redhat.cve:obj:20074559033"
  },
  "state": {}
}
